contenus/meta/meta_pied_de_page.md

```diff
@@ -1,3 +1,13 @@
+<div id="entretiens-utilisateurs" class="conseil">
+
+#### Aidez-nous à améliorer MesConseilsCovid
```

Comment:
⚠️ a11y: le H4 c'est pas cohérent avec la hiérarchie de titres de la page d'accueil, peut-être aussi sur d'autres pages.

https://addons.mozilla.org/fr/firefox/addon/headingsmap/ peut aider à vérifier rapidement.

Reply:
Œil de lynx, merci ! On met quoi du coup ?

Reply:
Si on n'est pas au même niveau pour chaque page ça va être pénible 😅

Reply:
> **Exception for fixed page sections**
> 
> In fixed sections of the page, for example in sidebars, the heading ranks should not change depending on the ranks in the content area. In those cases, consistency across pages is more important.

https://www.w3.org/WAI/tutorials/page-structure/headings/#exception-for-fixed-page-sections

Reply:
Du coup, je serais d’avis de mettre un <h2>